src/ProjectNeon/Assets/Features/Effects/OnNextCardEffect.cs

```diff
@@ -0,0 +1,32 @@
+﻿
+/**
+ * Wraps the execution of an effect so it is only good to execute after the next card is played.
+ * 
+ * Turn ending makes this effect active. 
+ */
+public class OnNextCardEffect : Effect
```

Comment:
This isn't quite what is needed for Astromancer. 

We can about the Astromancer's next card, not any party next card.

Reply:
@SilasReinagel I chose to make a generic effect, so we can reuse it later if needed.